src/partials/hero.html

```diff
@@ -1 +1,15 @@
-  
\ No newline at end of file
+<section class="hero-section container">
+  <h1 class="title-hero">Hello.<br>I’m Fullstack developer Lloyd Jefferson.</h1>
+
+  <nav class="soc-media">
+    <ul class="social-list">
+      <li class="social-item"><a class="item-soc">IG</a></li>
+      <li class="social-item"><a class="item-soc">FB</a></li>
+      <li class="social-item"><a class="item-soc">YT</a></li>
+      <li class="social-item"><a class="item-soc">GH</a></li>
+    </ul>
+    <address class="mail-addres">
+      <a href="mailto:lloydjefferson@gmail.com" class="mail-link">lloydjefferson@gmail.com</a>
+    </address>
+  </nav>
```

Comment:
Із тегу nav слід прибрати `address`, так як адреса не є навігацією по сайту
```suggestion
  <nav class="soc-media">
    <ul class="social-list">
      <li class="social-item"><a class="item-soc">IG</a></li>
      <li class="social-item"><a class="item-soc">FB</a></li>
      <li class="social-item"><a class="item-soc">YT</a></li>
      <li class="social-item"><a class="item-soc">GH</a></li>
    </ul>
    </nav>
      <address class="mail-addres">
      <a href="mailto:lloydjefferson@gmail.com" class="mail-link">lloydjefferson@gmail.com</a>
    </address>

```